casper/src/main/scala/coop/rchain/casper/Casper.scala

```diff
@@ -67,8 +67,7 @@ trait MultiParentCasper[F[_]] extends Casper[F, IndexedSeq[BlockHash]] {
   // this initial fault weight combined with our fault tolerance threshold t.
   def normalizedInitialFault(weights: Map[Validator, Long]): F[Float]
   def lastFinalizedBlock: F[BlockMessage]
-  // TODO: Refactor hashSetCasper to take a RuntimeManager[F] just like BlockStore[F]
-  def getRuntimeManager: F[Option[RuntimeManager[F]]]
+  def getRuntimeManager: F[RuntimeManager[F]]
```

Comment:
Changing the API for getRuntimeManager should have been in a different commit but I hope that isn't too much extra to swallow.